src/soc/rockchip/rk3399.c

```diff
@@ -312,7 +315,7 @@ struct layout_t * rk3399GetLayout(int i, int* mapping) {
 
 	pin = &rk3399->layout[mapping[i]];
 	if(pin->bank < 0 || pin->bank >= GPIO_BANK_COUNT) {
-		wiringXLog(LOG_ERR,"pin->bank out of range: %i, expect 0~4", pin->bank);
+		wiringXLog(LOG_ERR, "pin->bank out of range: %i, expect 0~4", pin->bank);
```

Comment:
Line 323 has the same issue:
`wiringXLog(LOG_ERR,"Pin %i is mapped to undefined pin on the hardware", i);`